### PR TITLE
(DOCS-5455) Add multi-sub management instructions

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Add multi-subscription management information.

### Motivation
DOCS-5455

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/guide/azure-portal.md [hunk 1]
```diff
@@ -91,14 +92,24 @@ Select **Change Plan** on the overview page to change your Datadog billing plan.
 The portal retrieves all the available Datadog plans for your tenant, including any private offers. Select the appropriate plan and click **Change Plan**.
 
 ## Datadog org configurations
+
+### Monitored subscriptions
+
+Select **Monitored Subscriptions** in the left sidebar to view or modify the scope of the Datadog resource. A list of currently monitored subscriptions appears. Use this view to configure the scope of the Datadog resource to monitor as many subscriptions as desired.  The subscription with the Datadog resource must be included in the scope.
```
Comment (hestonhoffman) [inline content/en/integrations/guide/azure-portal.md:98]:
```suggestion
Select **Monitored Subscriptions** in the left sidebar to view or modify the scope of the Datadog resource. A list of currently monitored subscriptions appears. Use this view to configure the scope of the Datadog resource to monitor as many subscriptions as desired. The subscription with the Datadog resource must be included in the scope.
```

File: content/en/integrations/guide/azure-portal.md [hunk 2]
```diff
@@ -91,14 +92,24 @@ Select **Change Plan** on the overview page to change your Datadog billing plan.
...
+
+   - To add subscriptions to monitor, click `+ Add Subscriptions`. The list of subscriptions available only includes subscriptions to which you have `Owner` role assigned. Select the subscriptions you wish to monitor and click `Add`.
```
Comment (hestonhoffman) [inline content/en/integrations/guide/azure-portal.md:100]:
```suggestion
   - To add subscriptions to monitor, click `+ Add Subscriptions`. The list of subscriptions available only includes subscriptions to which you have the `Owner` role assigned. Select the subscriptions you wish to monitor and click `Add`. 
```

File: content/en/integrations/guide/azure-portal.md [hunk 3]
```diff
@@ -91,14 +92,24 @@ Select **Change Plan** on the overview page to change your Datadog billing plan.
 The portal retrieves all the available Datadog plans for your tenant, including any private offers. Select the appropriate plan and click **Change Plan**.
 
 ## Datadog org configurations
+
+### Monitored subscriptions
+
+Select **Monitored Subscriptions** in the left sidebar to view or modify the scope of the Datadog resource. A list of currently monitored subscriptions appears. Use this view to configure the scope of the Datadog resource to monitor as many subscriptions as desired.  The subscription with the Datadog resource must be included in the scope.
+
+   - To add subscriptions to monitor, click `+ Add Subscriptions`. The list of subscriptions available only includes subscriptions to which you have `Owner` role assigned. Select the subscriptions you wish to monitor and click `Add`. 
+   - To remove subscriptions from being monitored with Datadog, select the subscriptions you wish to remove and click `Remove Subscriptions`. You can only remove subscriptions to which you have `Owner` role assigned.
```
Comment (hestonhoffman) [inline content/en/integrations/guide/azure-portal.md:101]:
```suggestion
   - To remove subscriptions from being monitored with Datadog, select the subscriptions you wish to remove and click `Remove Subscriptions`. You can only remove subscriptions to which you have the `Owner` role assigned.
```
Small suggestion. I generally avoid "of which / to which" constructions. They're a little formal. Perhaps: "Only accounts with the `Owner` role can remove subscriptions."

**Review by hestonhoffman — APPROVED** [review]
Couple small edits / suggestions

**Review by hestonhoffman — APPROVED** [review]
:shipit:

**Review by steveharrington1 — APPROVED** [review]

**Pull request merged**